yb-voyager/src/tgtdb/conn_pool.go

```diff
@@ -121,6 +126,29 @@ func (pool *ConnectionPool) getNextUriIndex() int {
 }
 
 func (pool *ConnectionPool) setSessionVars(conn *pgx.Conn) error {
+	sessionVarsPath := filepath.Join("/", "etc", "ybSessionVariables.sql")
```

Comment:
I would suggest the following:
- Replace the `SessionVars` map from the ConnectionPoolParams with a `SessionInitScript []string`.
- Rename `ConnectionPool.setSessionVars()` as `ConnectionPool.initSession()`.
- The `initSession()` simply runs the list of instructions specified in the `pool.params.SessionInitScript`.
- The importData() path where we instantiate the ConnectionPool, should read the `/etc/yb-voyager/sessionInitScript.sql` into a list of lines and pass it in the `ConnectionPoolParams`.

Reply:
The latest patch addresses this, please do review